### ISSUE TITLE
Suggestion: SMS commands permission

### ISSUE THREAD

**Issue by briggz84** [tananaev/traccar#3299]
Not sure if this is already available - a selectable permission check box in server settings which disables/enables users from sending device commands via SMS.

We currently have SMS available for certain important notifications, but are finding some users are sending unnecessary commands via SMS which is running up the bill.

Thanks

**Comment by tananaev** [comment]
I am thinking that we should implement something more generic. For example, we can have limits for incoming and outgoing SMS per user. This will help in cases when user gets too many notifications as well.

**Comment by jon-stumpf** [comment]
Having limits per user would be nice but having the ability to choose which users can send SMS is something that I would like as well.  My preference would be for a User Manager to control which users can send SMS.

**Comment by tananaev** [comment]
I don't see the difference. Limit is more generic. You can always set it to 0 to disable SMS for a particular user. Why would we need another separate flag for disabling SMS?

**Comment by jon-stumpf** [comment]
I did not think about setting the limit to zero.  And, the User Manager would only have this ability to set the limits, correct?

**Comment by tananaev** [comment]
I guess it makes sense to allow manager to configure it.

**Comment by briggz84** [comment]
I agree that the idea of SMS limitation would be beneficial with the added benefit of a 0 value disabling that user from any SMS. However, how would this effect SMS commands and SMS notifications separately? Is this what we define as incoming and outgoing SMS?

**Comment by tananaev** [comment]
Yes, I think those should be separate.

**Comment by SebastEnn** [comment]
Command limitation was implemented. Issue can be closed.

**Comment by tananaev** [comment]
This is not implemented yet.
